ICA/auto_mycosnp_ICA.py

```diff
@@ -0,0 +1,187 @@
+                sample_sheet.append(tmp)
+            if i.split('_')[5].split('.')[0] == '1':
+                tmp=i.split()[1].split('.')[0]+','+i.split()[1]
+                tmp=tmp+','+i.split()[1].replace('1','2')+'\n'
+                sample_sheet.append(tmp)
+
+        file = open("./%s_samplesheet/samplesheet.csv" % Experiment, "w")
+        file.write("".join(sample_sheet))
+        file.close()
+
+        file = open("./%s_samplesheet/samples.csv" % Experiment, "w")
+        file.write("".join(ica_ids))
+        file.close()
+
+        bashCommand = "icav2 projects enter Testing"
+        process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+        process.communicate()
+
+    #All samples must be linked to Testing project in order to run in the pipeline
+        sams=open('./%s_samplesheet/samples.csv' % Experiment,'r').read()
+        for i in range(len(sams.split(','))):
+               bashCommand = "icav2 projectdata link %s" % sams.split(',')[i]
+               process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+               process.communicate()
+
+
+        bashCommand = "icav2 projectdata upload ./%s_samplesheet" % Experiment
+        process = subprocess.Popen(bashCommand.split(" "), stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+        process.communicate(input=b'y')
+
+        bashCommand = "icav2 projectdata list --data-type FILE --parent-folder /%s_samplesheet/ --file-name samplesheet.csv" % Experiment
+        tmp= icav_out(bashCommand)
+        samplesheet=tmp[0].split(' ')[4]
+
+        bashCommand = "icav2 projectdata list --parent-folder /Candida_auris_clade_controls/Raw/"
+        tmp= icav_out(bashCommand)
+
+        controls=''
+        for i in tmp:
+            controls=controls+i.split()[4]+','
+
+        # Create icav2 argument
+        arg = "icav2 projectpipelines start nextflow CDC_mycosnp_V1_3 --input input_samples:"
+        arg = arg + controls
+        arg = arg + sams[:-1]
+        arg = arg + ' --input input_csv:%s ' % samplesheet
+        arg = arg + '--input modules_json:fil.59d555e385bc44373e0508da602cff6c --input schema_json:fil.67a59465102f49653e0b08da602cff6c --input project_dirs:fol.9f7bb648f8b14ad1bd5908da602d9944,fol.1c6b76e2679e46a5bd5708da602d9944,fol.4c3975313355455e3e0708da602cff6c,fol.df431f633c8a405b3db608da602cff6c,fol.a0755943991b4b713d9f08da602cff6c,fol.287f4196a2574a77346f08da603e530e,fol.de2553cf82f74a103d8c08da602cff6c,fol.8bb8690a65d8459e3d5608da602cff6c,fol.08cbd7c3121a4b503dd008da602cff6c --input fasta:fil.a582dffcf93846d6f72908da19bdaa25 '
+        arg = arg + '--user-reference CDC_mycosnp_V1_3_%s_update' % Experiment
+
+        print(arg)
+        process = subprocess.Popen(arg.split(" "), stdout=subprocess.PIPE)
+        process.communicate()
+
+        run=list(set(runs_on_ica)-set(runs_completed_analysis))
+        os.mkdir('/Volumes/IDGenomics_NAS/ICA/Runs_On_ICA/%s' % run[0] )
```

Comment:
Once this is clia validated, it would be awesome if it would to into /Volumes/IDGenomics_NAS/ICA/Runs_On_ICA or a directory for C. auris